docs/source/getting-started/installation.rst

```diff
@@ -23,7 +23,7 @@ Install SkyPilot using pip:
 
           pip install skypilot-nightly
 
-          # Or, choose an extra (defaults to [aws]):
+          # Or, choose an extra:
```

Comment:
Do we expect installing `pip install skypilot-nightly` alone is useful? Feels like the most common case for people is to directly go to an extra. If that's the case, maybe remove the no-extra case and hint "Choose your cloud"?

```
To use a subset of the clouds, combine some of the pip extras above to
reduce the dependencies:
```
may also need to change, e.g., "To use a few clouds, combine the pip extras:"

Reply:
Good call! Updated. PTAL